### ISSUE TITLE
How add more DEX exchanges?

### ISSUE THREAD

**Issue by CryptorClub** [AlgoNetwork/UniInch#1]
Hello. Its a great project and a nice start!
Would be appreciated, if you can suggest how to add more exchanges (add fee)
I see needed to create smart contract.

**Comment by AlgoNetwork** [comment]
Yes, If there is a demand, it can be deployed to other blockchains.

**Comment by CryptorClub** [comment]
> Yes, If there is a demand, it can be deployed to other blockchains.
Thanks for reply.
Can I ask about source of sol smart contract?

Thank you!
